app/src/main/res/layout/fragment_settings.xml

```diff
@@ -19,57 +19,44 @@
             android:layout_height="wrap_content"
             android:orientation="vertical">
 
-            <LinearLayout
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:orientation="vertical">
-
-                <androidx.appcompat.widget.SwitchCompat
-                    android:id="@+id/switchShareLocation"
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:gravity="center_vertical"
-                    android:padding="@dimen/padding_big"
-                    android:text="@string/shareLocation" />
+            <!-- Allgemeines
```

Comment:
I'd change the order of the links based on their importance:

1. Account bearbeiten
2. Impressum
3. Datenschutz
4. ---------------------
5. Abmelden
6. App Version